src/firefly/java/edu/caltech/ipac/firefly/server/visualize/VisServerOps.java

```diff
@@ -1434,9 +1437,66 @@ public static WebPlotResult getDS9Region(String fileKey) {
             retval = createError("on getDSRegion", null, e);
         }
         return retval;
+    }
 
+    private static final Map<String, String> footprintMap;
+    static {
```

Comment:
you and I and @loitly should talk about where the right place to store these files are.  I don't know if we want put it in 'visualize/reousrces'